src/plugins/webPage/components/WebPage.vue

```diff
@@ -1,16 +1,28 @@
 <template>
 <div class="l-iframe abs">
-    <iframe :src="currentDomainObject.url"></iframe>
+    <iframe :src="url"></iframe>
 </div>
 </template>
 
 <script>
+const sanitizeUrl = require("@braintree/sanitize-url").sanitizeUrl;
+
 export default {
     inject: ['openmct', 'domainObject'],
     data: function () {
         return {
             currentDomainObject: this.domainObject
         };
+    },
+    computed: {
+        url() {
+            let url = this.currentDomainObject.url;
+            if (url) {
```

Comment:
same as above, check if this works as expected sanitizeUrl(this.currentDomainObject.url);